src/runtime_src/core/pcie/driver/windows/include/XoclUser_INTF.h

```diff
@@ -201,7 +209,7 @@ typedef enum _XOCL_STAT_CLASS {
     XoclStatIpLayout,
     XoclStatKds,
     XoclStatKdsCU,
-
+    XoclStatRomInfo
```

Comment:
Change is not allowed.

Reply:
This class will be used in retrieving feature rom information through XOCL_IOCTL_STAT ioctl. So, added this here.

Reply:
This change is already part of device driver change. Please check https://github.com/Xilinx/MS_XOCLUSER/blob/master/Src/Inc/XoclUser_INTF.h#L204